devise.rb

```diff
@@ -57,22 +57,20 @@
   <% if notice %>
     <div class="alert alert-info alert-dismissible fade show m-1" role="alert">
       <%= notice %>
-      <button type="button" class="close" data-dismiss="alert" aria-label="Close">
-        <span aria-hidden="true">&times;</span>
+      <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close">
       </button>
     </div>
   <% end %>
   <% if alert %>
     <div class="alert alert-warning alert-dismissible fade show m-1" role="alert">
       <%= alert %>
-      <button type="button" class="close" data-dismiss="alert" aria-label="Close">
-        <span aria-hidden="true">&times;</span>
+      <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close">
       </button>
     </div>
   <% end %>
 HTML
 
-run 'curl -L https://github.com/lewagon/awesome-navbars/raw/master/templates/_navbar_wagon.html.erb > app/views/shared/_navbar.html.erb'
+run 'curl -L https://raw.githubusercontent.com/lewagon/awesome-navbars/fbdfb7685050dca29daa0ba4a65668cb6346d34a/templates/_navbar_wagon.html.erb > app/views/shared/_navbar.html.erb'
```

Comment:
Replacing the commit link by a branch link so that we fetch the latest version of the bootstrap 5 navbar here:
```suggestion
run 'curl -L https://raw.githubusercontent.com/lewagon/awesome-navbars/bootstrap-5/templates/_navbar_wagon.html.erb > app/views/shared/_navbar.html.erb'
```